### ISSUE TITLE
how to process the csv data provided in kaggle

### ISSUE THREAD

**Issue by AbigSnail** [caishiqing/multimodal-recommendation-kaggle#1]
hi~Could you tell me why do you do this in model.py file ?   logits = tf.matmul(state_seq, item_vectors, transpose_b=True)
